logconsumer_test.go

```diff
@@ -340,13 +339,11 @@ func TestContainerLogWithErrClosed(t *testing.T) {
 	}
 	// Simulate a transient closed connection to the docker daemon
 	i, _, err := dind.Exec(ctx, append([]string{"iptables", "-A"}, iptableArgs...))
-	if err != nil || i > 0 {
-		t.Fatalf("Failed to close connection to dind daemon: i(%d), err %v", i, err)
-	}
+	require.Zerof(t, i, "Failed to close connection to dind daemon: i(%d), err %v", i, err)
```

Comment:
suggestion: checks are back to front, NoError should be first, more below and above.